scripts/exgdas_global_marine_analysis_prep.py

```diff
@@ -5,7 +5,7 @@
 # Script name:         exufsda_global_marine_analysis_prep.py
 # Script description:  Stages files and generates YAML for UFS Global Marine Analysis
 #
-# Author: Guillaume Vernieres      Org: NCEP/EMC     Date: 2022-03-28
+# Author: Guillaume Vernieres      Org: NCEP/EMC     Date: 2022-03-23
```

Comment:
Not why it's edited, but we might as well just remove the header entirely. We'll sort out what to do when moving to the g-w.

Reply:
That's a glitch on my part. I'll take it out.